src/black/lines.py

```diff
@@ -684,18 +684,14 @@ def _maybe_empty_lines(self, current_line: Line) -> Tuple[int, int]:
                 return 0, 1
             return before, 1
 
+        # In preview mode, always allow blank lines, except right before a function
```

Comment:
Btw a few lines up we have `and current_line.is_triple_quoted_string`. Should that also use `is_docstring`? Seems preferable to have consistent behaviour for single quote docstrings as well

Reply:
Yeah I don't really understand why we sometimes use one and sometimes the other. I'll merge this first and then maybe make another preview style change harmonizing between these two.